src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultProjectSnapshotWorker.cs

```diff
@@ -63,7 +63,7 @@ public override Task ProcessUpdateAsync(ProjectSnapshotUpdateContext update, Can
             var configuration = await _configurationFactory.GetConfigurationAsync(update.UnderlyingProject);
             update.Configuration = configuration;
 
-            var result = await _tagHelperResolver.GetTagHelpersAsync(update.UnderlyingProject);
+            var result = await _tagHelperResolver.GetTagHelpersAsync(update.UnderlyingProject, CancellationToken.None);
```

Comment:
I think this is right. If we end up needing cancellation here we'll have to do work to support it, but now all of the signatures are correct 👍